### PR TITLE
chore(migrate to angular10): changes (DSP-1415)

### PR DESCRIPTION
**Pull request by waychal**
resolves DSP-1415

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,30 +43,30 @@
     "d3": "^5.15.1",
     "d3-force-3d": "^2.1.0",
     "dat.gui": "^0.7.7",
-    "jdnconvertiblecalendar": "^0.0.5",
-    "jdnconvertiblecalendardateadapter": "^0.0.13",
+    "jdnconvertiblecalendar": "^0.0.6",
+    "jdnconvertiblecalendardateadapter": "^0.0.15",
```
Comment (tobiasschweizer) [inline package.json:47]: you have to use 0.0.14 with Angular 10, 0.0.15 is already for Angular 11

Comment (tobiasschweizer) [reply package.json:47]: sorry, 0.0.15 is fine

**Review by tobiasschweizer — COMMENTED** [review]
I spotted a few warnings about modules built with commonJS:

> WARNING in dsp-app/node_modules/@dasch-swiss/dsp-js/test/data/api/admin/mock-users.js depends on 'json2typescript'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

> WARNING in dsp-app/node_modules/three-forcegraph/dist/three-forcegraph.module.js depends on 'ngraph.graph'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

> WARNING in dsp-app/node_modules/three-forcegraph/dist/three-forcegraph.module.js depends on 'ngraph.forcelayout'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

> WARNING in dsp-app/node_modules/three-forcegraph/dist/three-forcegraph.module.js depends on 'tinycolor2'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Maybe for some of these packages there is a newer version that could be used instead?

**Comment by waychal** [comment]
> I spotted a few warnings about modules built with commonJS:
> 
> > WARNING in dsp-app/node_modules/@dasch-swiss/dsp-js/test/data/api/admin/mock-users.js depends on 'json2typescript'. CommonJS or AMD dependencies can cause optimization bailouts.
> > For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
> 
> > WARNING in dsp-app/node_modules/three-forcegraph/dist/three-forcegraph.module.js depends on 'ngraph.graph'. CommonJS or AMD dependencies can cause optimization bailouts.
> > For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
> 
> > WARNING in dsp-app/node_modules/three-forcegraph/dist/three-forcegraph.module.js depends on 'ngraph.forcelayout'. CommonJS or AMD dependencies can cause optimization bailouts.
> > For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
> 
> > WARNING in dsp-app/node_modules/three-forcegraph/dist/three-forcegraph.module.js depends on 'tinycolor2'. CommonJS or AMD dependencies can cause optimization bailouts.
> > For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
> 
> Maybe for some of these packages there is a newer version that could be used instead?

I will check it.

**Comment by tobiasschweizer** [comment]
> > I spotted a few warnings about modules built with commonJS:
> > > WARNING in dsp-app/node_modules/@dasch-swiss/dsp-js/test/data/api/admin/mock-users.js depends on 'json2typescript'. CommonJS or AMD dependencies can cause optimization bailouts.
> > > For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
> > 
> > 
> > > WARNING in dsp-app/node_modules/three-forcegraph/dist/three-forcegraph.module.js depends on 'ngraph.graph'. CommonJS or AMD dependencies can cause optimization bailouts.
> > > For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
> > 
> > 
> > > WARNING in dsp-app/node_modules/three-forcegraph/dist/three-forcegraph.module.js depends on 'ngraph.forcelayout'. CommonJS or AMD dependencies can cause optimization bailouts.
> > > For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
> > 
> > 
> > > WARNING in dsp-app/node_modules/three-forcegraph/dist/three-forcegraph.module.js depends on 'tinycolor2'. CommonJS or AMD dependencies can cause optimization bailouts.
> > > For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
> > 
> > 
> > Maybe for some of these packages there is a newer version that could be used instead?
> 
> I will check it.

otherwise these warnings can be ignored, see https://stackoverflow.com/questions/62589229/angular-10-upgrade-fix-commonjs-or-amd-dependencies-can-cause-optimization-bai

**Review by flavens — APPROVED** [review]

**Comment by waychal** [comment]
@tobiasschweizer I have removed the warnings for CommonJS or AMD dependencies. Can you please review the PR again?

**Review by tobiasschweizer — APPROVED** [review]

**Pull request merged**